### PR TITLE
Datetime removal 

### PR DESCRIPTION
**Pull request by vn-ki**
Just some more datetime removals.

### CODE REVIEW

File: sunpy/coordinates/tests/test_frameattributes.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def test_on_frame(input):
 
 
 def test_non_string():
-    output = datetime.datetime.now()
+    output = parse_time(datetime.datetime.now())
```
Comment (Cadair) [inline sunpy/coordinates/tests/test_frameattributes.py]: should this not be `parse_time("now")`?

File: sunpy/coordinates/tests/test_frames.py [hunk 2]
```diff
@@ -209,11 +210,11 @@ def test_wrapping_off():
 def test_HEE_creation():
     # Smoke test to make sure HEE constructors work fine
     _ = HeliographicStonyhurst(lon=0*u.deg, lat=90*u.deg,
-                               obstime=datetime(2018, 12, 21))
+                               obstime=parse_time(('2018-12-21')))
```
Comment (Cadair) [inline sunpy/coordinates/tests/test_frames.py]: why the double bracket?

File: sunpy/coordinates/tests/test_frameattributes.py (hunk 1 above)
Comment (vn-ki) [reply sunpy/coordinates/tests/test_frameattributes.py]: That gives `utcnow` instead of `now`

File: sunpy/coordinates/tests/test_frames.py (hunk 2 above)
Comment (vn-ki) [reply sunpy/coordinates/tests/test_frames.py]: `parse_time` takes a tuple

File: sunpy/coordinates/tests/test_frameattributes.py (hunk 1 above)
Comment (Cadair) [reply sunpy/coordinates/tests/test_frameattributes.py]: thats probably more correct anyway

**Comment by Cadair** [comment]
huh, how did the astropy-time job fail because a remote file was missing?!

**Pull request merged**